### DISCUSSION TITLE
GraphRag Acceleratorデプロイ　十分なvCPUクォータ数について

### DISCUSSION THREAD

**Discussion by KonnToshinobu** [Azure-Samples/graphrag-accelerator#189] [General]
GraphRag Acceleratorをデプロイするためには、サブスクリプションにどのくらいのvCPUクォータが必要ですか。
現状は、10個のｖCPUです。
下記のエラーメッセージが出ています。
===Not enough Standard DSv5 Family vCPU quota for deployment. ===

**Comment by workwise1988** [comment]
Hi, Same with me, seeing below error message when I try to deploy the GraphRAG Accelerator to Azure
**Not enough Standard DSv5 Family vCPU quota for deployment. At least 12 vCPU is required.**

@KonnToshinobu , Is this issues resolved, if yes, can you please share how many vCPU's it worked with?
